locksmith/src/models/index.ts

```diff
@@ -7,6 +7,7 @@ import { Lock } from './lock'
 import { Transaction } from './transaction'
 import { AuthorizedLock } from './authorizedLock'
 import { EventLink } from './eventLink'
+import { LockMetadata } from './lockmetadata'
```

Comment:
Let's keep naming consistent please ;)

```suggestion
import { LockMetadata } from './lockMetadata'
```